package.json

```diff
@@ -50,12 +50,12 @@
     "underscore": "~1.8.3"
   },
   "devDependencies": {
-    "babel-core": "6.17.0",
+    "babel-core": "^6.24.0",
```

Comment:
`"babel-core": "6.24.0",`
better to keep exact version instead of feature. In the past, testing and builds were inconsistent from feature updates.
same for the other changes